governance-exercises/DAO Masons poll-exercise-04.md

```diff
@@ -0,0 +1,58 @@
+
+I could then spend the remaining time waiting for the execs to be ready 'coordinating an emergency governance call to discuss solutions and a plan for subsequent actions, explicitly inviting known stakeholders' as dictated by MIP24c3 [Emergency Response Procedure](https://mips.makerdao.com/mips/details/MIP24#MIP24c3).
+
+This reduces the time that we're signaling to the world that there is a vulnerability in Maker and reduces risk. They mentioned that the exploit can't be used in 'current circumstances' but it's unclear how dynamic those circumstances are and it's possible we could be attacked if we signal we are vulnerable. Technically the vulnerability is already signaled by the forum post so avoiding the poll only reducing the broadcast.
+
+I would verify with Imogen and Geneveive, but I'm assuming that if Fix B goes live the threat becomes slightly more acute, as an attacker could use the fix to discern the nature of the remaining vulnerability and exploit it, so if Fix A is happening it should happen asap. I want to verify this and add it as a response to the forum thread.
+
+For the purpose of this exercise I'll assume they reveal some information that makes me think a poll is a good idea again. The most likely justification being that having a poll open allows more people to have a chance to vote, and then the executive can be a shorter window to execute it quickly when it's ready.
+
+Since the fixes will be ready in 8 hours, I would set the poll to 7 hours as the poll shouldn't slow down the emergency response. I'll still coordinate an emergency governance call to discuss solutions.
+
+I'd work to get the poll live asap, and hopefully be able to delegate to someone on my team to begin reaching out to every known stakeholder. We see that the governance/delegates folder in the community repo has handles for members. We'd spend the rest of the time before the poll ends to reach out to everybody we can and alert them that there is an emergency vote in progress.
+
+### Draft any forum responses required or implied by your course of action.
+
+As Governance Facilitator I am processing a request from Protocol Facilitator Imogen to declare this an Urgent or Emergency situation and will refer to [MIP24 Emergency Response](https://mips.makerdao.com/mips/details/MIP24#MIP24c3) as appropriate.
+
+I'm taking the following actions:
+
+1. Drafting an emergency poll
+2. Coordinating an emergency governance call to discuss solutions and a plan for subsequent actions
+
+I have some clarifying questions for Imogen and Geneveive and will ask them publicly to help inform the voters in the poll.
+
+1. MKR holders will want to understand the risk to their funds. Please expand on the statement "Fix A is likely to incur losses to MKR holders of approximately 20 million DAI over two weeks."
```

Comment:
So this is maybe more obvious with more context, but in this case I mean that ~20M Dai would be lost from the surplus buffer. The buffer is generally considered to be the funds owned by the protocol, and therefore indirectly by MKR holders. 

Fair point though, that I could have been more explicit in the exercise.